libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -44,54 +44,66 @@ export class SkyActionButtonContainerComponent
    */
   @Input()
   public set alignItems(value: SkyActionButtonContainerAlignItemsType) {
-    this._alignItems = value;
+    this.#_alignItems = value;
   }
 
   public get alignItems(): SkyActionButtonContainerAlignItemsType {
-    return this._alignItems || 'center';
+    return this.#_alignItems || 'center';
   }
 
   @ContentChildren(SkyActionButtonComponent)
-  private actionButtons: QueryList<SkyActionButtonComponent>;
+  private actionButtons: QueryList<SkyActionButtonComponent> | undefined;
```

Comment:
Content children and view children should be changed to `public`.

Reply:
done